zetaclient/context/app.go

```diff
@@ -81,14 +64,17 @@ func (a *AppContext) Config() config.Config {
 
 // GetBTCChainAndConfig returns btc chain and config if enabled
 func (a *AppContext) GetBTCChainAndConfig() (chains.Chain, config.BTCConfig, bool) {
-	btcConfig, configEnabled := a.Config().GetBTCConfig()
-	btcChain, _, paramsEnabled := a.GetBTCChainParams()
+	cfg, configEnabled := a.Config().GetBTCConfig()
```

Comment:
Here is place for future refactor you can track

1. Get rid of `GetBTCChainAndConfig` and `GetSolanaChainAndConfig` and use one single method similar to below to return a deep copy of ChainParams for chain bootstrap or chain parameter updating. Not for this PR.

`GetExternalChainParams(chainID int64) (observertypes.ChainParams, bool)`

Reply:
Correct, we can resolve everything based on `chainID` + `IsEVMChain()`, `IsUTXOChain()`, `IsSolanaChain()`, ...